### DISCUSSION TITLE
HTTPException when trying to create a forum thread

### DISCUSSION THREAD

**Discussion by stonymf** [Rapptz/discord.py#9185] [Q&A] [answered]
Hi, I can't seem to figure out how to have my bot create a forum post / thread. I've confirmed that my channel exists and the channel object has the create_thread() method. This is the method to make a new forum post, correct?

I've tried various combinations of arguments and keyword arguments. 

This my line currently:
`thread = await channel.create_thread(name="new forum thread")`

And I get this response:
`discord.errors.HTTPException: 400 Bad Request (error code: 50006): Cannot send an empty message`

Does anyone have a confirmed working example of a bot successfully creating a forum thread? I'd be grateful if you could share it, thanks!

**Comment by LeoCx1000** [comment, marked as answer]
If you look at the documentation, you will notice that it has parameters like .send does. This is to create the starter message of the forum. You didn't pass any of those parameters.

[``ForumChannel.create_thread``](https://discordpy.readthedocs.io/en/stable/api.html#discord.ForumChannel.create_thread)

**Reply by stonymf** [reply]
Hey thanks! I don't see any parameters there that would correspond to starter message contents.

**Reply by Rapptz** [reply]
There's parameter information for content, embeds, files, etc. in that documentation.

**Reply by stonymf** [reply]
You were right, the content parameter was required. The docs & code had it as optional, so I opened a PR: https://github.com/Rapptz/discord.py/pull/9186
